great_expectations/expectations/core/expect_column_value_lengths_to_be_between.py

```diff
@@ -215,7 +219,7 @@ def _atomic_prescriptive_template(
             at_least_str, at_most_str = handle_strict_min_max(params)
 
             if params["mostly"] is not None:
-                params["mostly_pct"] = num_to_str(
+                params_with_json_schema["mostly_pct"]["value"] = num_to_str(
```

Comment:
Just to confirm, this is the one Expectation  where we were modifying the incorrect `params` dictionary?